pkg/limatype/lima_yaml.go

```diff
@@ -20,7 +20,7 @@ type LimaYAML struct {
 	OS                 *OS           `yaml:"os,omitempty" json:"os,omitempty" jsonschema:"nullable"`
 	Arch               *Arch         `yaml:"arch,omitempty" json:"arch,omitempty" jsonschema:"nullable"`
 	Images             []Image       `yaml:"images,omitempty" json:"images,omitempty" jsonschema:"nullable"`
-	// Deprecated: Use VMOpts.Qemu.CPUType instead.
+	// Deprecated: Use VMOpts.QEMU.CPUType instead.
```

Comment:
Soon:
```suggestion
	// Deprecated: Use VMOpts["qemu"].CPUType instead.
```

Reply:
I used the yaml names since they are less ugly, but that is the "after" state and this was the "before" state...